Product/Production/Services/DocumentRetrieveCore/src/test/java/gov/hhs/fha/nhinc/docretrieve/nhin/proxy/NhinDocRetrieveProxyWebServiceSecuredImplTest.java

```diff
@@ -99,15 +99,15 @@ public void testGuidanceg0() throws Exception {
         NhinDocRetrieveProxyWebServiceSecuredImpl impl = getImpl();
         NhinTargetSystemType target = getTarget("1.1", "3.0");
         impl.respondingGatewayCrossGatewayRetrieve(request, assertion, target, GATEWAY_API_LEVEL.LEVEL_g0);
-        verify(cache).getEndpointURL(anyString(), anyString(), any(UDDI_SPEC_VERSION.class));
+        verify(cache).getEndpointURL(anyString(), anyString(), any(UDDI_SPEC_VERSION.class), anyString());
     }
 
     @Test
     public void testGuidanceg1() throws Exception {
         NhinDocRetrieveProxyWebServiceSecuredImpl impl = getImpl();
         NhinTargetSystemType target = getTarget("1.1", "3.0");
         impl.respondingGatewayCrossGatewayRetrieve(request, assertion, target, GATEWAY_API_LEVEL.LEVEL_g1);
-        verify(cache).getEndpointURL(anyString(), anyString(), any(UDDI_SPEC_VERSION.class));
+        verify(cache).getEndpointURL(anyString(), anyString(), any(UDDI_SPEC_VERSION.class), anyString());
```

Comment:
Same here, and in the test above Mockito.isA() for the UDDI class

Reply:
We can revisit it and update the complete test class in a separate ticket.